[Cache] KB Migration (#3073)
cloudflare/cloudflare-docs: products/cache/src/content/about/cache-analytics.md

## Review cache status

The **Requests summary** graph depicts how your traffic changes over time, such as in response to a high-traffic event or a recent configuration change. Note that the Requests summary content is based on a 10% sample of requests.

**Served by Cloudflare** indicates content served by Cloudflare that did not require contacting your origin web server.  **Served by Origin** indicates traffic served from the origin web server. 

For **Data Transfer**, **Revalidated** requests are considered **Served by Cloudflare**. However, revalidated requests count as **Served by Origin** within the **Requests **view. This analytics behavior reflects that Cloudflare must check the origin web server for revalidated cache requests before returning a result from cache.

**Cache status** graphs help explain why traffic is served from Cloudflare versus the origin web server. The graph shows analytics by content-type to portray how different components of your website perform:

For a list of cache statuses and their descriptions, refer to [Cloudflare cache responses](/about/default-cache-behavior#cloudflare-cache-responses).

## Review requests by source

> Cache Analytics shows top metrics (Top-N) for several request components. Apply filters before reviewing Top-N metrics. For example, filtering to only view traffic with an Expired or Revalidated Cache status allows review of which URLs were primarily responsible for those statuses.

Reviewer: ```suggestion
Cache Analytics shows top metrics (Top-N) for several request components. Apply filters before reviewing Top-N metrics. For example, filtering to only view traffic with an Expired or Revalidated Cache status lets you review which URLs were primarily responsible for those statuses.
```
